eth/contracts/Advisory.sol

```diff
@@ -0,0 +1,41 @@
+// Most Advisory
```

Comment:
Request: we don't need this on eth, remove.

Reply:
Done